x-pack/plugins/security/server/authorization/reset_session_page.test.tsx

```diff
@@ -28,6 +28,25 @@ describe('ResetSessionPage', () => {
         logoutUrl="/path/to/logout"
         buildNumber={100500}
         basePath={mockCoreSetup.http.basePath}
+        customBranding={{}}
+      />
+    );
+
+    expect(body).toMatchSnapshot();
+  });
+
+  it('renders as expected with custom page title', async () => {
```

Comment:
__nit:__ same comment here, you might need to change the test to test something else instead of `title`.